administrator/components/com_associations/views/association/view.html.php

```diff
@@ -139,9 +143,9 @@ protected function addToolbar()
 		JToolbarHelper::apply($associatedView . '.apply');
 		JToolbarHelper::save($associatedView . '.save');
 		JToolbarHelper::save2new($associatedView . '.save2new');
-		JToolbarHelper::cancel($associatedView . '.cancel', 'JTOOLBAR_CLOSE');
+		JToolbarHelper::cancel('association.cancel', 'JTOOLBAR_CLOSE');
```

Comment:
nice. the Close button works fine now!
But attention, you need to checkout the items too on close.